translations/de.ts

```diff
@@ -1132,12 +1132,12 @@ instead of system taskbar.</source>
         <location filename="../src/widget/form/settings/generalsettings.ui" line="370"/>
         <source>Always notify about new messages in groupchats.</source>
         <comment>toolTip for Group chat always notify</comment>
-        <translation>Immer auf neue Nachrichten in Gruppenchats hinweisen.</translation>
+        <translation>Du wirst benachrichtigt, wenn neue Nachrichten in Gruppenchats eintreffen.</translation>
```

Comment:
Was wäre mit: "Du wirst immer über neue Nachrichten in Gruppenchats benachrichtigt."